internal/operators/cnv/cnv_operator.go

```diff
@@ -160,3 +125,20 @@ func (o *operator) GetProperties() models.OperatorProperties {
 func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 	return &Operator
 }
+
+// GetHostRequirementsForRole provides operator's requirements towards host in a given role
+func (o *operator) GetHostRequirementsForRole(_ context.Context, _ *common.Cluster, role models.HostRole) (*models.ClusterHostRequirementsDetails, error) {
+	switch role {
+	case models.HostRoleMaster:
+		return &models.ClusterHostRequirementsDetails{
+			CPUCores: masterCPU,
+			RAMMib:   masterMemory,
+		}, nil
+	case models.HostRoleWorker, models.HostRoleAutoAssign:
```

Comment:
I would not use `HostRoleAutoAssign` here. We are unable to validate when auto role is provided.

Reply:
Changed in #1329